lib/src/query.c

```diff
@@ -165,6 +164,7 @@ typedef struct {
 } QueryState;
 
 typedef Array(TSQueryCapture) CaptureList;
+typedef Array(CaptureList) CaptureListPoolEntry;
```

Comment:
What do you think of eliminating this typedef? It seems like it's only used in one place; also the name suffix "Entry" makes me think that it's going to be represent one element of some collection, which isn't the case here.

Reply:
:+1: